features/assets.feature

```diff
@@ -18,5 +18,7 @@ Feature: Assets
   @app-authenticating-proxy
   Scenario: Check draft assets require authentication
```

Comment:
Should we match the naming convention of the scenario above as we're now checking we can preview the asset?

For example, 'Check a draft asset can be served'.

Reply:
Good point. I've amended with that 👍.